synopsys-task/src/synopsys-task/input.ts

```diff
@@ -265,7 +265,7 @@ export const COVERITY_LOCAL = getInput(
   constants.COVERITY_LOCAL_KEY_CLASSIC_EDITOR,
   constants.BRIDGE_COVERITY_LOCAL_KEY
 );
-export const COVERITY_AUTOMATION_PRCOMMENT = getInput(
+export const COVERITY_AUTOMATION_PRCOMMENT = getBoolInput(
```

Comment:
`getInput` method used to treat default false in the classic editor as a string, so the message for deprecated parameters was showing like this:
`[warning][bridge_coverity_automation_prcomment, bridge_blackduck_fixpr_enabled, bridge_blackduck_automation_prcomment] is/are deprecated.`

`getBoolInput()` resolves this issue.